pkg/util/util.go

```diff
@@ -53,6 +53,15 @@ func IsSRIOVVmi(vmi *v1.VirtualMachineInstance) bool {
 	return false
 }
 
+func IsVDPAVMI(vmi *v1.VirtualMachineInstance) bool {
```

Comment:
Nit: VDPAVMI is a bit difficult to read/understand right away. Following the convention of the above func `IsSRIOVVmi` would be a little easier to read.
```suggestion
func IsVDPAVmi(vmi *v1.VirtualMachineInstance) bool {
```

Reply:
@prnaraya Already fixed.